packages/flash-loans/src/aave/AaveCollateralSwapSdk.ts

```diff
@@ -462,6 +462,12 @@ export class AaveCollateralSwapSdk {
     ])
   }
 
+  readonly AAVE_DAPP_ID_PER_TYPE: Record<AaveFlashLoanType, string> = {
```

Comment:
Nitpick: it should be a static field or a const. No reason to make it an instance field